Code of Conduct.md

```diff
@@ -0,0 +1,18 @@
+## Contributor Code Of Conduct
+
+We as contributors pledge to respect all people who are engaged in this project. We are committed to making participation in our project a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+Some examples of unacceptable behavior by participants include:
+
+-	The use of sexualized language or imagery
+-	Insulting/derogatory comments
+-	Public or private harassment
+-	Publishing other's private information, such as physical or electronic addresses, without explicit permission
+
+Collaborators and project maintainers have the responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team. The project team will review the complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
Should we add one contact person's email address here?

Reply:
i saw that in one of the examples, but I don't know if it is required

Reply:
I guess we need any contact, otherwise there is no way people can privately contact us.

Reply:
ok, so we will need all members emails @jasmineortega @sy25wang

Reply:
my email: thomas.siu@gmail.com

Reply:
sy25wang@gmail.com

Reply:
jasmineortega29@gmail.com